easybuild/easyconfigs/d/DCMTK/DCMTK-3.6.3-GCCcore-7.3.0.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'CMakeMake'
+
+name = 'DCMTK'
+version = '3.6.3'
+
+homepage = 'https://dicom.offis.de/dcmtk'
+description = """DCMTK is a collection of libraries and applications implementing large parts the DICOM standard.
+It includes software for examining, constructing and converting DICOM image files, handling offline media, sending
+and receiving images over a network connection, as well as demonstrative image storage and worklist servers."""
+
+toolchain = {'name': 'GCCcore', 'version': '7.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://dicom.offis.de/download/dcmtk/dcmtk%s/' % version.replace('.', '')]
+sources = ['%(namelower)s-%(version)s.tar.gz']
+checksums = ['63c373929f610653f10cbb8218ec643804eec6f842d3889d2b46a227da1ed530']
+
+builddependencies = [
+    # use same binutils version that was used when building GCC toolchain
+    ('binutils', '2.30', '', True),
+    ('CMake', '3.11.4')
+]
+
+dependencies = [
+    ('zlib', '1.2.11'),
+    ('libjpeg-turbo', '2.0.0'),
+    ('LibTIFF', '4.0.9'),
+    ('libpng', '1.6.34'),
+    ('libxml2', '2.9.8'),
+    ('libiconv', '1.15'),
+]
+
+sanity_check_paths = {
+    'files': ['bin/dcmdump', 'bin/dcmj2pnm'],
+    'dirs': ['lib'],
+}
+
+moduleclass = 'bio'
```

Comment:
@crubb Hmm, how is this related to bioinformatics? Isn't this a more general tool?

I'd suggest `tools`, or maybe `vis`?